projects/workflows-creator/src/lib/builder/builder.component.ts

```diff
@@ -202,8 +202,31 @@ export class BuilderComponent<E> implements OnInit, OnChanges {
         });
       });
       this.updateDiagram();
+      this.hideElseBlockIfRequired();
     }
   }
+  /**
+   * This function checks if the else block should be hidden based on the type and number of events in
+   * the event group.
+   */
+  hideElseBlockIfRequired() {
+    const events = this.eventGroups[0].children;
+    let value = events[0].node.state.get('value');
```

Comment:
there is a possibility that this line will give error 
since the length check is later

Reply:
thr is always a  default node grp in this.eventGroups, it will never be empty , so no error will occur in future